### PR TITLE
Fix im2latex AC on windows

### PR DESCRIPTION
**Pull request by morkovka1337**
Some fixes to make AC for im2latex models work on windows correctly

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py [hunk 1]
```diff
@@ -171,8 +177,13 @@ def render_routine(line):
         if not os.path.exists(pdf_filename):
             logging.info('ERROR: %s cannot compile\n', file_idx)
         else:
-            subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
-                            pdf_filename, png_filename], check=True, stdout=PIPE, stderr=PIPE)
+            if os.name == 'nt':
+                convert_command = subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py:181]: please check command status and handle errors or change on check=True (it made this automatically)

File: tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py [hunk 2]
```diff
@@ -171,8 +177,13 @@ def render_routine(line):
...
+                            pdf_filename, png_filename],
+                     check=False, stdout=PIPE, stderr=PIPE, shell=True)
+            else:
+                convert_command = subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py:185]: the same comment about error check

File: tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py [hunk 3]
```diff
@@ -154,7 +157,10 @@ def render_routine(line):
     """
     formula, file_idx, folder_path = line
     output_path = os.path.join(folder_path, file_idx)
-    pre_name = output_path.replace('/', '_').replace('.', '_')
+    if os.name == 'nt':
+        pre_name = output_path.replace('\\', '_').replace('.', '_').replace(":", "_")
+    else:
+        pre_name = output_path.replace('/', '_').replace('.', '_')
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py:163]:
```suggestion
        pre_name = os.path.normcase(output_path).replace('/', '_').replace('.', '_')
```

File: tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py [hunk 4]
```diff
@@ -171,8 +177,13 @@ def render_routine(line):
...
         else:
-            subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
-                            pdf_filename, png_filename], check=True, stdout=PIPE, stderr=PIPE)
+            if os.name == 'nt':
+                subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
+                                                  pdf_filename, png_filename],
+                                                 check=True, stdout=PIPE, stderr=PIPE, shell=True)
+            else:
+                subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
+                                                  pdf_filename, png_filename], check=True, stdout=PIPE, stderr=PIPE)
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py:186]:
```suggestion
            subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
                                                  pdf_filename, png_filename],
                                                 check=True, stdout=PIPE, stderr=PIPE, shell=True)
```
shell=True works in the same way on win and linux, so it is not problem to run with it always like in other subprocess calls above

File: tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py (hunk 1 above)
Comment (morkovka1337) [reply tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py]: Added `shell=True`. It was changed on the debugging purpose, but later I forgot to change it back.

File: tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py (hunk 4 above)
Comment (morkovka1337) [reply tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py]: Merged commands

**Review by eaidova — APPROVED** [review]

**Pull request merged**